cfme/tests/services/test_iso_service_catalogs.py

```diff
@@ -104,7 +114,7 @@ def catalog():
 
 
 @pytest.yield_fixture(scope="function")
-def catalog_item(setup_provider, provider_crud, provider_type, provisioning, vm_name, dialog,
+def catalog_item(provider_init, provider_crud, provider_type, provisioning, vm_name, dialog,
```

Comment:
We should be using the setup_provider fixture wherever possible. In 2116, we have created a separate module scoped setup_provider fixture I think, I am trying to remove the these custome provider_init functions throughout the code. Maybe we'll just wait for this one till 2116?